setup.py

```diff
@@ -30,7 +30,8 @@
 release_status = "Development Status :: 5 - Production/Stable"
 dependencies = [
     "googleapis-common-protos >= 1.6.0, < 2.0dev",
-    "protobuf >= 3.12.0, < 3.18.0",
```

Comment:
This was originally added as python 2 users were getting unintended protobuf upgrades - see #276 and #277